src/core/reactor.cc

```diff
@@ -2569,7 +2569,8 @@ void reactor::register_metrics() {
             sm::make_current_bytes("total_memory", [] { return memory::stats().total_memory(); }, sm::description("Total memory size in bytes")),
             sm::make_current_bytes("allocated_memory", [] { return memory::stats().allocated_memory(); }, sm::description("Allocated memory size in bytes")),
             sm::make_counter("reclaims_operations", [] { return memory::stats().reclaims(); }, sm::description("Total reclaims operations")),
-            sm::make_counter("malloc_failed", [] { return memory::stats().failed_allocations(); }, sm::description("Total count of failed memory allocations"))
+            sm::make_counter("malloc_failed", [] { return memory::stats().failed_allocations(); }, sm::description("Total count of failed memory allocations")),
+            sm::make_counter("oversized_allocs", [] { return memory::stats().large_allocations(); }, sm::description("Total count of oversized memory allocations"))
```

Comment:
A side note: The memory::stats() API is quite terrible in a sense that it copies around ~100 bytes just get 8

Reply:
@amnonh - is that worth looking into?

Reply:
It's not that bad, copying 128 bytes is 8 instructions with AVX.

Moreover, the ABI is to pass the address of the return type as a hidden parameter, so typically it won't be copied.